packages/storybook/cxl-lumo-styles/icons-preview.js

```diff
@@ -0,0 +1,166 @@
+// @see: https://github.com/vaadin/docs/blob/838707b08368ca4a225e2e12840ff980b5ecc7ea/frontend/demo/foundation/icons-preview.ts
```

Comment:
Doc tags don't have punctuation like `:`

Reply:
File opening comment highly likely makes sense to have docblock, not line comment.